### PR TITLE
docs: update kratos video link

### PR DESCRIPTION
**Pull request by vinckr**
Merge when the video is released.

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security. vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [x] I have added tests that prove my fix is effective or that my feature
      works.
- [x] I have added or changed [the documentation](docs/docs).

### CODE REVIEW

File: docs/versioned_docs/version-v0.3/quickstart.mdx [hunk 1]
```diff
@@ -14,7 +14,7 @@ Please be aware that this guide is not a replacement for studying the docs. You
 must understand core concepts and APIs to use ORY Kratos productively. This is
 merely a guide to get you set up with some examples.
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/5t1Zr_zJc7E" frameborder="0" allowfullscreen></iframe>
+<iframe width="560" height="315" src="https://www.youtube.com/embed/JhlIZ3aChXc" frameborder="0" allowfullscreen></iframe>
```
Comment (aeneasr) [inline docs/versioned_docs/version-v0.3/quickstart.mdx:17]: But the video is not done with those versions, correct?

Comment (vinckr) [reply docs/versioned_docs/version-v0.3/quickstart.mdx:17]: Yes.

I can remove it from the older versions if that would be better

I could leave the old video in there as well, as it is only unlisted at the moment. 

But I would prefer to remove the old video, as it has the old logo and is just worse in general.

Comment (vinckr) [reply docs/versioned_docs/version-v0.3/quickstart.mdx]: Removed it from v0.3. 
v0.4 should be ok right?

Comment (aeneasr) [reply docs/versioned_docs/version-v0.3/quickstart.mdx]: You made the video ;)

**Pull request merged**